styles/books/english-composition/book.scss

```diff
@@ -12,33 +12,36 @@ $bandColor: #DCB83D;
 @import '../../designs/cosmos/pdf/folio';
 
 @include add_settings((
-  CultureAnnotation:(
+  ParaAnnotationWithIcon: (
+    _selectors: ('p.annotation'),
+  ),
+  CultureAnnotationIcon:(
```

Comment:
Do we want to keep the naming consistent and have it be [...]AnnotationWithIcon?

Reply:
Actually these are different kind of elements.

![image](https://user-images.githubusercontent.com/40228854/213494223-c9f1d705-27ad-47b2-8f66-80fa11b7e626.png)

`ParaAnnotationWithIcon` describes paragraph, `CultureAnnotationIcon`, `LinguisticAnnotationIcon` etc. describes just icons (from spans).

Reply:
Oh!!! I misread that, then great! No complaints!